tools/apollo/.shed.yml

```diff
@@ -3,7 +3,12 @@ categories:
 description: Access an Apollo instance from Galaxy
 long_description: |
     Talk to a (local) apollo server from Galaxy
-name: apollo
```

Comment:
@erasche I changed this shed.yml to gga, but of course we can switch back to erasche if you prefer (I didn't find these tools on MTS)

Reply:
nope, would rather things aren't under my namespace. they were never pushed to MTS, xref #5